src/common/websocket/index.ts

```diff
@@ -34,22 +33,59 @@ export class ApiManager {
         return ApiManager.instance
     }
 
+    public static readonly READY_STATE_KEY = 'websocket_ready_state'
+
+    private setReadyStateInSessionStorage(state: number) {
```

Comment:
hi @wojciech-deriv ,
actually i needed a global state in this call to check the ready_state of WS calls , and in deriv APP they are using MOBX to achieve this   since we don't have  global state like redux or mobx i used session storage.

Reply:
hi @wojciech-deriv changed  too static variable